### PR TITLE
[Corda-ENT] Add Networkmap service

### PR DESCRIPTION
**Pull request by abevers**
**Changelog**
- Add network map service Helm chart

**Reviewed by**
@suvajit-sarkar 

**Linked issue**
#778

### CODE REVIEW

File: platforms/corda-ent/charts/nmap/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v2
+appVersion: "1.2"
+description: A Helm chart for CENM Networkmap
+name: nmap
+version: "1.0.0"
```
Comment (sownak) [inline platforms/corda-ent/charts/nmap/Chart.yaml:5]: newline needed where there is this red sign

Comment (abevers) [reply platforms/corda-ent/charts/nmap/Chart.yaml]: Changed in commit [d778500](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/841/commits/d77850089dd8c95b11ae4d8e96b42480fd82c838)

File: platforms/corda-ent/charts/nmap/values.yaml [hunk 2]
```diff
@@ -0,0 +1,141 @@
+# Default values for the Networkmap Service for  Registration job.
```
Comment (suvajit-sarkar) [inline platforms/corda-ent/charts/nmap/values.yaml:1]: Should be just networkmap service

File: platforms/corda-ent/charts/nmap/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+                    break
+                  fi
+                  COUNTER=`expr "$COUNTER" + 1`
+              done
+
+              if [ "$COUNTER" -ge {{ $.Values.healthcheck.readinessthreshold }} ]
+              then
+                # printing number of trial done before giving up
+                echo "$COUNTER"
+                echo "ssl idman certificates might not have been put in vault.  Giving up!!!"
```
Comment (suvajit-sarkar) [inline platforms/corda-ent/charts/nmap/templates/deployment.yaml:135]: change idman to nms in code comments

File: platforms/corda-ent/charts/nmap/templates/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,318 @@
...
+              COUNTER=1
+              while [ "$COUNTER" -lt {{ $.Values.healthcheck.readinessthreshold }} ]
+              do 
+                LOOKUP_SECRET_RESPONSE=$(curl -sS --header "X-Vault-Token: ${VAULT_TOKEN}" ${VAULT_ADDR}/v1/${CERTS_SECRET_PREFIX}/{{ $.Values.notaryName }}/nodeInfo | jq -r 'if .errors then . else . end')
+                if echo ${LOOKUP_SECRET_RESPONSE} | grep "errors"
+                  then
+                    sleep {{ $.Values.healthcheck.readinesscheckinterval }}
+                  else
+                  notary_nodeinfo=$(echo ${LOOKUP_SECRET_RESPONSE} | jq -r '.data["nodeInfoFile"]')
+                  echo "${notary_nodeinfo}" | base64 -d > ${MOUNT_PATH}/notary-nodeinfo/node_info_file
```
Comment (suvajit-sarkar) [inline platforms/corda-ent/charts/nmap/templates/deployment.yaml:106]: use the full name of nodeInfo file

**Comment by suvajit-sarkar** [comment]
@abevers Please incorporated the review comments in a separate PR for the same issue

**Pull request merged**